src/status_im/multiaccounts/create/core.cljs

```diff
@@ -224,50 +222,65 @@
           :else {:db (assoc-in db [:intro-wizard :step]
                                (inc-step step))})))
 
+(defn get-selected-multiaccount [db]
+  (let [{:keys [selected-id multiaccounts]} (:intro-wizard db)]
+    (some #(when (= selected-id (:id %)) %) multiaccounts)))
+
+(defn prepare-default-account [{:keys [publicKey address]}]
+  {:public-key publicKey
+   :address    address
+   :color      colors/blue
+   :default?   true
+   :name       "Status account"})
+
 (fx/defn on-multiaccount-created
-  [{:keys [signing-phrase
-           status
-           db] :as cofx}
-   {:keys [pubkey address mnemonic installation-id
-           keycard-instance-uid keycard-key-uid keycard-pairing keycard-paired-on] :as result}
-   password
-   {:keys [seed-backed-up? login? new-multiaccount?] :or {login? true}}]
-  (let [normalized-address (utils.hex/normalize-hex address)
-        multiaccount            {:public-key             pubkey
-                                 :installation-id        (or installation-id (get-in db [:multiaccounts/new-installation-id]))
-                                 :address                normalized-address
-                                 :name                   (gfycat/generate-gfy pubkey)
-                                 :status                 status
-                                 :signed-up?             true
-                                 :desktop-notifications? false
-                                 :photo-path             (identicon/identicon pubkey)
-                                 :signing-phrase         signing-phrase
-                                 :seed-backed-up?        seed-backed-up?
-                                 :mnemonic               mnemonic
-                                 :keycard-instance-uid   keycard-instance-uid
-                                 :keycard-key-uid        keycard-key-uid
-                                 :keycard-pairing        keycard-pairing
-                                 :keycard-paired-on      keycard-paired-on
-                                 :settings               (constants/default-multiaccount-settings)
-                                 :syncing-on-mobile-network? false
-                                 :remember-syncing-choice? false
-                                 :new-multiaccount?           new-multiaccount?}]
-    (when-not (string/blank? pubkey)
+  [{:keys [signing-phrase db] :as cofx} _ password {:keys [seed-backed-up? login?] :or {login? true}}]
+  (let [{:keys [id mnemonic] :as multiaccount} (get-selected-multiaccount db)
+        {:keys [publicKey address]} (get-in multiaccount [:derived constants/path-whisper-keyword])
+        default-wallet-account (get-in multiaccount [:derived constants/path-default-wallet-keyword])
+        {:networks/keys [networks]} db
+        new-multiaccount            {;;multiaccount
+                                     :id                         id
```

Comment:
it seems we don't need this id

Reply:
it should be address :( something like root-address

Reply:
yeah in any case it won't matter with status-go in the future because we will receive the list of multiaccount at startup and then will login using multiaccount struct